boards/groups.inc.mk

```diff
@@ -0,0 +1,116 @@
+BOARD_GROUP.arm7 := \
+  avsextrem \
+  msba2 \
+  #
+
+BOARD_GROUP.avr8 := \
```

Comment:
Would it make sense to change this (and the others as well) to `BOARD_GROUP += ...`? This way one could add temporary elements from the command line, possibly useful when playing with new boards?

Reply:
yup!

Reply:
done